c182t.xml

```diff
@@ -449,8 +449,8 @@
                     <max>20</max>
                 </domain>
                 <range>
-                    <min>1</min>
-                    <max>-1</max>
+                    <min>-1</min>
+                    <max>1</max>
                 </range>
```

Comment:
@HHS81 I cannot see changes here.
What is affected by this property?

@Zaretto I think the original idea was to have the right aileron negative, when the left one is positive, and vice-versa.

Reply:
It is indeed like you said- right aileron negative etc... I borrowed the code from the c172p

Reply:
my min/max changes were wrong- caused because I wasn't paying enough attention and I had clipto errors because of an old version of c182s.xml JSBSim on the path.

Now reverted; the rest of the pull request is still valid and should be reviwed.

Sorry for the noise.